checkov/terraform/context_parsers/tf_plan/tf_plan_json.py

```diff
@@ -354,16 +277,126 @@ class CfnJSONDecoder(json.JSONDecoder):
 
     def __init__(self, *args, **kwargs):
         json.JSONDecoder.__init__(self, *args, **kwargs)
-        self.parse_object = cfn_json_object
+        self.parse_object = self.cfn_json_object
         self.parse_array = self.JSONArray
         self.parse_string = py_scanstring
         self.memo = {}
         self.object_pairs_hook = check_duplicates
         self.scan_once = py_make_scanner(self)
+        self.newline_indexes = []
+
+    def decode(self, s):
+        """Overridden to retrieve indexes """
+        self.newline_indexes = find_indexes(s)
+        obj = super(CfnJSONDecoder, self).decode(s)
```

Comment:
```suggestion
        obj = super().decode(s)
```
No need to write it here, it will call the correct super type `decode`.